pom.xml

```diff
@@ -1625,7 +1611,6 @@
           <version>2.43.0</version>
           <executions>
             <execution>
-              <phase>verify</phase>
```

Comment:
This isn't required because `spotless:check` is already bound to Maven's `verify` phase by default (see https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#binding-to-maven-phase).